server/services/source-map.js

```diff
@@ -200,18 +259,18 @@ fixSourcePath = function(source, nodeModulesBase, sourcesBase) {
 
 // Processes the source map (when exists) of an instrumented file to fix broken sources paths
 registerSourceMap = function(filepath) {
-  Log.time('START registerSourceMap', filepath);
   const sourceMapPath = filepath + '.map';
   let fileContent = parseJSON(sourceMapPath, true);
   if (fileContent) {
-    fileContent = _alterSourceMapPaths.call(this, fileContent,
+    Log.time('START registerSourceMap', filepath);
+    fileContent = alterSourceMapPaths.call(this, fileContent,
       filepath.startsWith('../web.browser/') || filepath.startsWith(abspath.clientSide));
     Log.info('Add source map for file', sourceMapPath);
     sourceMap.registerMap(filepath, fileContent);
+    Log.timeEnd('END registerSourceMap', filepath);
```

Comment:
The string passed to `console.time` and `console.timeEnd` needs to be the same. Otherwise the CI looks promising !